test/blackbox-tests/test-cases/coq/no-stdlib.t/run.t

```diff
@@ -0,0 +1 @@
+  $ dune build
```

Comment:
Could you also add a small explanation of the test above in a comment? (Anything you write without two spaces in a cram test is a comment).